### PR TITLE
Add server CLI integration test for entry commands

### LINKED ISSUE #1889: Add integration tests to exercise spire-server CLI commands
We don't really have integration tests that exercise the spire-server CLI commands. Add integration tests for them.

### PR DESCRIPTION
**Pull request by lsantire**
**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**

Server CLI integration test

**Description of change**

Add a new step to the _spire-server-cli_ test suite, in order to test the following commands:
- entry create
- entry update
- entry show
- entry delete

**Which issue this PR fixes**

It partially fixes #1889. Some tests for other commands are still pending.

### CODE REVIEW

File: test/integration/suites/spire-server-cli/03-entry [hunk 1]
```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+# Create bundles of federated tds to be used then
```
Comment (amartinezfayo) [inline test/integration/suites/spire-server-cli/03-entry:3]:
```suggestion
# Create bundles of federated trust domains to be used by other commands
```

**Review by amartinezfayo — COMMENTED** [review]
Thank you @lsantire for this!
I think that ideally we should try to exercise the different options of each command as much as possible. For example, creating / updating / showing entries with the `admin` or `downstream` bit set, specifying a DNS name, entry expiration, etc.
I'm OK taking this PR as is as a base to be improved in following PRs.
Thank you for this contribution!

I have one nit before merging this.

**Comment by lsantire** [comment]
Hi @amartinezfayo, thanks for your review!

> I think that ideally we should try to exercise the different options of each command as much as possible

Okay, I will take a look again at every option for the commands and I will improve this test.

**Review by amartinezfayo — APPROVED** [review]
Thank you @lsantire! 🎉

**Pull request merged**